src/js-form-validator.ts

```diff
@@ -356,4 +363,36 @@ export class FormValidator {
     }
     return status;
   }
+
+  private static HasValidLength(inputValue: string, minLength: number, maxLength: number): boolean {
+    if (!this.IsValidLengthValue(minLength) && !this.IsValidLengthValue(maxLength))
+      return true;
+
+    if (this.IsValidLengthValue(minLength) && inputValue.length < minLength ||
+        this.IsValidLengthValue(maxLength) && inputValue.length > maxLength)
+      return false;
+
+    return true;
+  }
+
+  private static IsValidLengthValue(value: number): boolean {
+    return !isNaN(value) && value !== 0;
+  }
+
+  private static GetInputLengthErrorMessage(inputValue: string, minLength: number, maxLength: number): string {
+    let errorMessage = '';
+
+    if (this.IsValidLengthValue(minLength) && inputValue.length < minLength)
+      errorMessage = `Entry must be greater than ${minLength} characters`;
+
+    if (!this.IsValidLengthValue(minLength) && this.IsValidLengthValue(maxLength) && inputValue.length > maxLength)
+      errorMessage = `Entry must be less than ${maxLength} characters`;
+
+    if (this.IsValidLengthValue(minLength) && this.IsValidLengthValue(maxLength) && inputValue.length > maxLength)
+      errorMessage += `${errorMessage} and less than ${maxLength} characters`;
+
+    return errorMessage.length > 0 ? `${errorMessage}.` : 'Error';
+  }
 }
+const digits = value.match(/\d+/g);
```

Comment:
Since you are making this a variable, it might help readability to make the other regular expression a named variable too.